libs/admin/src/lib/components/admin-page/admin-page.component.scss

```
@@ -1,5 +1,7 @@
 @use "../../styles/variables";
 
+$sub-navigation-bar-height: 84px;
+
 ::ng-deep .navigation .navigation__link {
```

`::ng-deep` without any prefix is essentially just global css. This was not added in this PR, but we might just as well fix it while we're here.
```suggestion
:host ::ng-deep .navigation .navigation__link {
```